### ISSUE TITLE
cannot run on Ubuntu 19.10

### ISSUE THREAD

**Issue by silviogames** [crashinvaders/gdx-texture-packer-gui#75]
when I try to run the launcher.sh contained in the .zip file downloaded from the releasepage the followed prompt occurs:

Inconsistency detected by ld.so: dl-lookup.c: 111: check_match: Assertion `version->filename == NULL || ! _dl_name_match_p (version->filename, map)' failed!

this is the java version:
openjdk 11.0.6 2020-01-14
OpenJDK Runtime Environment (build 11.0.6+10-post-Ubuntu-1ubuntu119.10.1)
OpenJDK 64-Bit Server VM (build 11.0.6+10-post-Ubuntu-1ubuntu119.10.1, mixed mode, sharing)

**Comment by metaphore** [comment]
There might be something won't with the local Java distribution. Have you tried to run JAR file directly using ‘java -jar ./gdx-texture-packer.jar‘?

**Comment by filosofis** [comment]
I got the same issue on Arch. Tried installing from aur. I also ran the jar file directly.
this is my java verison:
openjdk version "13.0.2" 2020-01-14
OpenJDK Runtime Environment (build 13.0.2+8)
OpenJDK 64-Bit Server VM (build 13.0.2+8, mixed mode)

**Comment by fgnm** [comment]
I have the same issue too. It's a Java related problem. It works if you downgrade to OpenJDK-8
```
sudo apt-get install openjdk-8-jdk
sudo update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64
```

Anyway I hope that developers could fix this problem with newer JDK versions.

**Comment by metaphore** [comment]
It looks like a problem with LWJGL2 LibGDX's backend, its the code base generally outdated and relies on some functionality that is outdated and banned in the modern JDK/JVM releases.
Migration to LWJGL3 is not that a big deal, but the problem is some parts of the app needs to be updated/reworked due to missing AWT implementation for LibGDX in that backend. I'll put it to the roadmap. This will take a while since I cannot spend too much time on this project right now, so I'd recommend to set up the environment to run the app through dedicated JRE (version < 9).

**Comment by byk0t** [comment]
> I have the same issue too. It's a Java related problem. It works if you downgrade to OpenJDK-8
> 
> ```
> sudo apt-get install openjdk-8-jdk
> sudo update-java-alternatives --set /usr/lib/jvm/java-1.8.0-openjdk-amd64
> ```
> 
> Anyway I hope that developers could fix this problem with newer JDK versions.

The same story. I had to downgrade from openjdk-14 to openjdk-8 on ubuntu 19.10

**Comment by metaphore** [comment]
The issue was fixed in the 4.9.0 release. Feel free to reopen if the problem persists.

**Comment by silviogames** [comment]
4.9.0 works perfectly now on Ubuntu!
